devops/scripts/asa/run_asa.sh

```diff
@@ -8,16 +8,33 @@ set -e
 # Tested and supported runtimes: podman, docker
 CONTAINER_RUNTIME="podman"
```

Comment:
can we have this resolved at runtime? eg. Use whichever is present podman/docker and report error if none?

Reply:
sorry just realized this wasn't even part of this changeset, im just thinking out loud here 😆

Reply:
Hehe, that's totally fine! By that I seen that we have podman set as default and it was supposed to be docker. I'll change that and will ask for re-approval.

I'll do runtime detection with priority to the first on the list in the follow up PR. That's not time consuming and rather straight forward.